Query01.php

```diff
@@ -11,123 +11,138 @@
 		<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
 		<link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
 
-
-		<meta name = "Keywords" content = "Graeme, Music Database, Soundwave, Music, Tunes"/>
-		<meta name = "Author" content = "Liv Amer"/>
-		<meta name = "Description" content = "Graeme's Music Database"/>
-		<meta name = "viewport" content = "width=device-width,initial-scale=1"/>
-
+		
+        <meta name = "Keywords" content = "Graeme, Music Database, Soundwave, Music, Tunes"/>
+        <meta name = "Author" content = "Liv Amer"/>
+        <meta name = "Description" content = "Graeme's Music Database"/>
+        <meta name = "viewport" content = "width=device-width,initial-scale=1"/>
+        
 
 	</head>
 	<body>
-
-		<!-- Navbar (sit on top) -->
+		
+			<!-- Navbar (sit on top) -->
 		<div class="w3-top">
 			<div class="w3-bar w3-white w3-wide w3-padding w3-card">
 				<img src="images/placeholder.jpg" width="40" height="40">
 
 				<a href="indexV1.php" class="w3-bar-item w3-button"><b>logo</b> </a>
 
+
 				<!-- Float links to the right. Hide them on small screens -->
-				<div class="w3-right w3-hide-small">	
-					<a href="indexV1.php" class="w3-bar-item w3-button">Home</a>
-					<a href="playlistV1.php" class="w3-bar-item w3-button">Playlist </a>	
-					<a href="contact.html" class="w3-bar-item w3-button" >Contact</a>
-					<a href="03_add_user(settings).php" class="w3-bar-item w3-button">Settings</a>	
+				<div class="w3-right w3-hide-small">
+					<a href="indexV1.php" class="w3-bar-item w3-button w3-xxlarge fa fa-home"></a>
+					<a href="playlistV1.php" class="w3-bar-item w3-button w3-xxlarge material-icons">playlist_add</a>	
+					<a href="contact.php" class="w3-bar-item w3-button w3-xxlarge material-icons" >contact_support</a>
+					<a href="signout.php" class="w3-bar-item w3-button w3-xxlarge material-icons">logout</a>
 				</div>
 			</div>
 		</div>
 
 
+		
 		<br>
 		<br>
 		<br>
 		<br>	
-
+		
 		<!-- Container Queries -->
-		<div class="w3-content w3-container w3-padding-64 w3-center w3-white" id="about">
-			<a href="playlist_genre.php"> <img src="images/placeholder.jpg" class="w3-button w3-round w3-opacity w3-hover-opacity-off"  alt="Genre A - Z" height="170" style="width:310px"> </a>
+		<div class="w3-content w3-container w3-padding-64 w3-center w3-yellow w3-image">
```

Comment:
changed colours and to image so it is responsive; reason being i can see which is what container and the width of it